### DISCUSSION TITLE
Cancellare un alunno

### DISCUSSION THREAD

**Discussion by gaiaianni** [iisgiua/giuaschool#790] [Richieste di aiuto]
Buon pomeriggio.
Come posso eliminare un alunno?
É possibile solo disabilitarlo?
 
![Screenshot 2024-11-26 at 14-16-57 Modifica Alunni](https://github.com/user-attachments/assets/b72c3575-2274-4dd0-ac6a-104db8cf7628)

**Comment by trinko** [comment]
Non si possono cancellare gli utenti, ma solo disabilitarli.
La cancellazione degli utenti disabilitati avviene in automatico al cambio di anno scolastico.
